e2e/keywords/utility.resource

```diff
@@ -0,0 +1,28 @@
+*** Settings ***
+Documentation     
```

Comment:
```suggestion
Documentation
```

Reply:
Have removed the redundant blank. Thanks.